README.md

```diff
@@ -55,3 +55,13 @@ This book is generated with [Pandoc book template](https://github.com/wikiti/pan
 ## License
 
 [Creative Commons Attribution 4.0 International](https://creativecommons.org/licenses/by/4.0/)
+
+Animtion is gives life to design, Here are some best resources I can find on the internet.
```

Comment:
This is great, thank you!

Just a suggestion, is it possible to add a heading and fix the small typo in `Animtion`? :)